diff --git a/node_modules/@types/react-dom/index.d.ts b/node_modules/@types/react-dom/index.d.ts
index d9a3f7d..a6a264a 100755
--- a/node_modules/@types/react-dom/index.d.ts
+++ b/node_modules/@types/react-dom/index.d.ts
@@ -29,7 +29,7 @@ import {
 export function findDOMNode(instance: ReactInstance | null | undefined): Element | null | Text;
 export function unmountComponentAtNode(container: Element | DocumentFragment): boolean;
 
-export function createPortal(children: ReactNode, container: Element, key?: null | string): ReactPortal;
+export function createPortal(children: ReactNode, container: Element | DocumentFragment, key?: null | string): ReactPortal;
 
 export const version: string;
 export const render: Renderer;
